approved.json

```diff
@@ -152,29 +152,37 @@
     "G8IH_v4_64"
   ],
   "azuremssqlmanaged": [
-    "Gen5_v4",
-    "Gen5_v8",
-    "Gen5_v16",
-    "Gen5_v24",
-    "Gen5_v32",
-    "Gen5_v40",
-    "Gen5_v64",
-    "Gen5_v80",
-    "G8IM_v4",
-    "G8IM_v8",
-    "G8IM_v16",
-    "G8IM_v24",
-    "G8IM_v32",
-    "G8IM_v40",
-    "G8IM_v64",
-    "G8IM_v80",
-    "G8IH_v4",
-    "G8IH_v8",
-    "G8IH_v16",
-    "G8IH_v24",
-    "G8IH_v32",
-    "G8IH_v40",
-    "G8IH_v64"
+		"GP_G8IM_v4",
```

Comment:
Please update the SKUs based on https://github.com/intel/terraform-intel-azure-mssql-managed-instance/blob/main/policies.md